### PR TITLE
Update core to v0.2.3

### PR DESCRIPTION
**Pull request by tigrannajaryan**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,6 +24,11 @@ ci: all test-with-cover
 	$(MAKE) -C testbed install-tools
 	$(MAKE) -C testbed runtests
 
+.PHOYN: precommit
```
Comment (pjanotti) [inline Makefile]: typo

**Review by pjanotti — COMMENTED** [review]
There is a typo on the Makefile but other than that LGTM

File: Makefile (hunk 1 above)
Comment (tigrannajaryan) [reply Makefile]: Fixed.

**Review by pjanotti — APPROVED** [review]
👍

**Review by owais — APPROVED** [review]

**Pull request merged**